### PR TITLE
Changelog for 1.12.0

### PR DESCRIPTION
**Pull request by rgrunber**
Signed-off-by: Roland Grunberg <rgrunber@redhat.com>

### CODE REVIEW

**Comment by fbricon** [comment]
Please add https://github.com/redhat-developer/vscode-quarkus/pull/558

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,58 @@
...
+ * Support for the `config_ordinal` property in `microprofile-config.properties`. See [eclipse/lsp4mp#289](https://github.com/eclipse/lsp4mp/issues/289).
+ * Property evaluation should support the environment variable default value notation. See [eclipse/lsp4mp#241](https://github.com/eclipse/lsp4mp/issues/241).
+ * Display property value when hovering over a key that isn't defined in the application. See [eclipse/lsp4mp#285](https://github.com/eclipse/lsp4mp/issues/285).
+ * REST client code lens only shows up for `GET` annotations. See [eclipse/lsp4mp#94](https://github.com/eclipse/lsp4mp/issues/94)
+ * JAXRS code lens URL should always appear above method declaration. See [eclipse/lsp4mp#194](https://github.com/eclipse/lsp4mp/issues/194)
+ * Support `microprofile-health` 3.0 and later. See [eclipse/lsp4mp#314](https://github.com/eclipse/lsp4mp/issues/314)
+ * `@ConfigProperties` validation should check the annotation's fully qualified name. See [eclipse/lsp4mp#304](https://github.com/eclipse/lsp4mp/issues/304).
+ * Disable JVM logging by default to avoid language server failure. See [#548](https://github.com/redhat-developer/vscode-quarkus/issues/548).
```
Comment (datho7561) [inline CHANGELOG.md:52]: If we want to include all the bug fixes from lsp4mp, here's also `eclipse/lsp4mp#295`, `eclipse/lsp4mp#311`, `eclipse/lsp4mp#315`, `eclipse/lsp4mp#325`

**Review by datho7561 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (rgrunber) [reply CHANGELOG.md:52]: I left out those since they seemed pretty specific to lsp4mp, or didn't really fix something that would have been user-visible.

Comment (datho7561) [reply CHANGELOG.md:52]: I looked through them, and you're right, they're mostly not applicable to the end user. Sorry about that. I think we should include https://github.com/eclipse/lsp4mp/issues/325 though, it has to do with the snippet value.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,59 @@
...
+ * Clickable inlay hint for Java type in Qute templates. See [#533](https://github.com/redhat-developer/vscode-quarkus/pull/533).
+ * Validation, completion & definition support for enum in `#switch` section. See [quarkus-ls#689](https://github.com/redhat-developer/quarkus-ls/issues/689), [#690](https://github.com/redhat-developer/quarkus-ls/issues/690), [quarkus-ls#691](https://github.com/redhat-developer/quarkus-ls/issues/691).
+  * Display property expression evaluation as inlay hint. See [vscode-microprofile#108](https://github.com/redhat-developer/vscode-microprofile/pull/108).
+ * Added Qute code actions for similar text suggestions. See [quarkus-ls#602](https://github.com/redhat-developer/quarkus-ls/issues/602).
+ * Code actions to create Java field / getter method / template extension. See [quarkus-ls#536](https://github.com/redhat-developer/quarkus-ls/issues/536), [quarkus-ls#676](https://github.com/redhat-developer/quarkus-ls/issues/676), [quarkus-ls#677](https://github.com/redhat-developer/quarkus-ls/issues/677).
+ * Display Javadoc on hover for property/method . See [#452](https://github.com/redhat-developer/quarkus-ls/issues/452).
```
Comment (datho7561) [inline CHANGELOG.md]: `quarkus-ls#425`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,59 @@
...
+
+### Performance
+ * Use `resolve` for code action in Qute templates. See [quarkus-ls#604](https://github.com/redhat-developer/quarkus-ls/issues/604).
+ * Improve code action performance with `CodeAction#data` & `resolveCodeAction`. See [vscode-microprofile#124](https://github.com/redhat-developer/vscode-microprofile/pull/124), [eclipse/lsp4mp#171](https://github.com/eclipse/lsp4mp/issues/171).
+ * Activation trigger should be more selective. See [quarkus-ls#497](https://github.com/redhat-developer/vscode-quarkus/issues/497).
+ * Detect cyclical class hierarchy when calculating all fields and methods of an object. See [quarkus-ls#725](https://github.com/redhat-developer/quarkus-ls/issues/725).
+ * Fix cancel support with `CompletableFuture` compose. See [quarkus-ls#679](https://github.com/redhat-developer/quarkus-ls/pull/679).
+
+### Bug Fixes
+ * Duplicate template data when using type safe expressions. See [#quarkus-ls750](https://github.com/redhat-developer/quarkus-ls/issues/750).
```
Comment (datho7561) [inline CHANGELOG.md]: Move the `#` in the link text (eg. `#quarkus-ls750` --> `quarkus-ls#750`)

**Review by datho7561 — CHANGES_REQUESTED** [review]
Sorry, I missed two small typos when I last reviewed

File: CHANGELOG.md (hunk 2 above)
Comment (rgrunber) [reply CHANGELOG.md]: Done. Got a bit lost when looking for 425 instead of 452 :P

**Review by datho7561 — APPROVED** [review]
Thanks Roland!

**Pull request merged**